### ISSUE TITLE
Project browser result set records should link to drupal project summary page

### ISSUE THREAD

**Issue by zednis** [tetherless-world/dco-elasticsearch#23]
The previous S2S based project browser had links to the project summary page (a drupal page).  This should be added as the href on the project title.

**Comment by mrpatrickwest** [comment]
All we need to do is to link to the page

https://deepcarbon.net/dco_project_summary?uri=the_uri

We already have the project's URI. Not the DCO-ID, but the URI.

**Comment by am-e** [comment]
I've made the change and fixed the title sorting issue.
Should we have a not indexed version of the title?
Where should I push the changes, project-browser branch?

**Comment by zednis** [comment]
the project-browser branch has been merged.  Create a new branch off master.

I don't think it would hurt to have a non-analyzed version of the title, the code would look like this:

```
"title": {
        "type": "multi_field",
          "fields": {
            "title": {
              "type": "string",
              "index": "analyzed"
            },
            "exact": {
              "type": "string",
              "index": "not_analyzed"
            }
          }
      },
```

note the inner fields are named "title" and "exact".

**Comment by am-e** [comment]
Merged #24
